reactive/etcd.py

```diff
@@ -505,6 +506,30 @@ def read_tls_cert(cert):
         return data
 
 
+@when('nrpe-external-master.available')
+@when_not('nrpe-external-master.initial-config')
+def initial_nrpe_config(nagios=None):
+    set_state('nrpe-external-master.initial-config')
+    update_nrpe_config(nagios)
+
+
+@when('etcd.installed')
+@when('nrpe-external-master.available')
+@when_any('config.changed.nagios_context',
+          'config.changed.nagios_servicegroups')
+def update_nrpe_config(unused=None):
+    # List of systemd services that will be checked
+    services = ('etcd',)
+
+    # The current nrpe-external-master interface doesn't handle a lot of logic,
+    # use the charm-helpers code for now.
+    hostname = nrpe.get_nagios_hostname()
+    current_unit = nrpe.get_nagios_unit_name()
+    nrpe_setup = nrpe.NRPE(hostname=hostname, primary=False)
+    nrpe.add_init_service_checks(nrpe_setup, services, current_unit)
```

Comment:
A "very permissive reactive clause" ? AIUI, it will only fire when the config option "nagios_context" or "nagios_servicegroups" is changed, which shouldn't happen all that often. Is that not the case ?